sentencepiece/yum/sentencepiece.spec

```diff
@@ -0,0 +1,85 @@
+Name:		sentencepiece
+Version:	0.1.83
+Release:	1%{?dist}
+Summary:	An unsupervised text tokenizer for Neural Network-based text generation
+
+Group:		Applications/Text
+License:	Apache-2.0
+URL:		https://github.com/google/sentencepiece
+Source0:	https://github.com/google/sentencepiece/releases/download/v%{version}/sentencepiece-%{version}-Source.tar.xz
+
+BuildRoot:      %{_tmppath}/%{name}-%{version}-%{release}-%(%{__id_u} -n)
+BuildRequires:	cmake3
+BuildRequires:	pkgconfig
+BuildRequires:	gperftools-devel
+Requires:	%{name}-libs = %{version}-%{release}
+
+%description
+The SentencePiece is an unsupervised text tokenizer for Neural Network-based
+text generation.
+
+%package libs
+Summary:	Runtime libraries for SentencePiece
+Group:		System Environment/Libraries
+License:	Apache-2.0
+Requires(post):	/sbin/ldconfig
+Requires(postun):	/sbin/ldconfig
+
+%description libs
+This package contains the libraries for SentencePiece
+
+%package tools
+Summary:	Tools for SentencePiece
+Group:		Applications/Text
+License:	Apache-2.0
+Requires:	%{name}-libs = %{version}-%{release}
+
+%description tools
+This package contains tools for manipulate models for SentencePiece
+
+%package devel
+Summary:	Libraries and header files for SentencePiece
+Group:		Development/Libraries
+Requires:	%{name}-libs = %{version}-%{release}
+
+%description devel
+This package contains header files to develop a software using SentencePiece
+
+%prep
+%setup -q -n %{name}-%{version}-Source
+
+%build
+%cmake3 . -DCMAKE_INSTALL_LIBDIR=lib64
+make %{?_smp_mflags}
+
+%install
+rm -rf $RPM_BUILD_ROOT
+make install DESTDIR=$RPM_BUILD_ROOT INSTALL="install -p"
```

Comment:
Why is `INSTALL` needed?

Reply:
Thanks, it is no need to do it, I've removed.